src/modules/recipes/NewRecipeForm/index.jsx

```diff
@@ -0,0 +1,226 @@
+import { useForm, useFieldArray } from "react-hook-form";
+import { useRecipesDispatch } from "../RecipesProvider";
```

Comment:
```suggestion
import { useRecipesDispatch, RECIPE_ACTIONS } from "../RecipesProvider";
```

then remove the next line